lib/till.rb

```diff
@@ -0,0 +1,59 @@
+require 'json'
+
+class Till
+
+  attr_accessor :customer_order
+  attr_accessor :receipt
+  attr_accessor :customer_bill
+  attr_accessor :tax
+  attr_accessor :total
```

Comment:
Ok, but usually you'd probably not repeat `attr_accessor` over and over: you can write this on one line separating your accessors with commas. 

Also, needing this many public accessors is generally a code smell - do other classes really need to be able to read and write all these properties?